app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/react/ManifestEvents.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { get } from 'lodash';
+import { IManifest } from '@spinnaker/core';
+
+interface IManifestEventsProps {
+  manifest: IManifest;
+}
+
+export class ManifestEvents extends React.Component<IManifestEventsProps> {
+  private pillStyle(e: any): string {
```

Comment:
Events (https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/domain/IManifest.ts#L13) could be typed.